pkg/apitype/updates.go

```diff
@@ -70,6 +70,12 @@ type UpdateMetadata struct {
 	Environment map[string]string `json:"environment"`
 }
 
+// LocalPolicyPack contains information about the locally specified Policy Packs ran with
+// an update.
+type LocalPolicyPack struct {
+	Path string `json:"path"`
```

Comment:
Maybe include a note about the path being potentially truncated?

Reply:
We dont actually use this anymore, this was part of my initial work. So I am going to remove it instead.